pep-0692.rst

```diff
@@ -118,7 +125,7 @@ type ``int``). This indicates that the function should be called as follows::
 Inside the function itself, the type checkers should treat
 the ``kwargs`` parameter as a ``TypedDict``::
```

Comment:
Should type checkers only do this when TypedDicts are wrapped with `Unpack`? I would also say "annotation" consistently rather than "parameter".

Reply:
It would seem that they are in fact referring to the type of the `kwargs` dict itself, rather than the type of its annotation object, though arguably that would be the collected argument values rather than the parameter name.

Reply:
I just went with `When using Unpack, the type checkers should treat kwargs inside the
function body as a TypedDict`, should be understandable. If I remember all the argument vs parameter definitions correctly, `parameter` here seemed ok - when doing static analysis this would be still a parameter (e.g. while reading the function's body as a human)